### ISSUE TITLE
Multi classes with custom number of kpt, under oks loss

### ISSUE THREAD

**Issue by axhionning** [ruiz-manuel/yolov7-pose-custom#6]
This is my oks_loss part , and I also change all the kpt numbers'part ,why my kpt_loss didn't converge？Is anyone else experiencing the same problem? How did you fix it?

# # oks based loss
                    d = (pkpt_x-tkpt[i][:,0::2])**2 + (pkpt_y-tkpt[i][:,1::2])**2
                    s = torch.prod(tbox[i][:,-2:], dim=1, keepdim=True)
                    kpt_loss_factor = (torch.sum(kpt_mask != 0) + torch.sum(kpt_mask == 0))/(torch.sum(kpt_mask != 0)+1e-9)
                    lkpt += kpt_loss_factor*((1 - torch.exp(-d/(s*(4*sigmas**2)+1e-9)))*kpt_mask).mean()

appreciate all replies!
